functions/feeds.php

```diff
@@ -44,7 +44,7 @@ static protected function __new_feed( $url ) {
 		$content   = get_site_transient( $cache_key );
 
 		if ( $content === False ) {
-			$content = @file_get_contents( $url );
+			$content = @wp_remote_get( $url );
```

Comment:
Since we're transitioning to `wp_remote_get()`, which returns a `WP_Error` object on failure, we should probably check if `wp_remote_get()` returns a `WP_Error` instead of `False` on line 48, in lieu of using PHP's error suppression (`@`).

e.g.
```php
$content = wp_remote_get( $url );
if ( is_wp_error( $content ) ) {
    $failed = true;
    // ...
}
```